packages/grafana-data/src/types/theme.ts

```diff
@@ -241,6 +241,15 @@ export interface GrafanaTheme extends GrafanaThemeCommons {
     formCheckboxBgChecked: string;
     formCheckboxBgCheckedHover: string;
     formCheckboxCheckmark: string;
+    formRadioButtonText: string;
+    formRadioButtonTextActive: string;
+    formRadioButtonTextHover: string;
+    formRadioButtonBg: string;
+    formRadioButtonBgDisabled: string;
+    formRadioButtonBgActive: string;
+    formRadioButtonBorder: string;
+    formRadioButtonBorderHover: string;
+    formRadioButtonBorderActive: string;
```

Comment:
supprised that we need so many new theme variables here. Is there a way to reduce these, and create more high level form color variables (border, highlight, hover etc). 

the alternative is to create local variables for these inside getRadioButtonStyles, not sure adding so many component-specific variables to the theme is the way to go. I remember we discussed this earlier in the year. Think some high-level form variables that multiple components can re-use is a good idea, but this feels a bit too much and component specific. 

thoughts?

Reply:
Wasn't too specific about this, but majority of these variables are to be moved to the corresponding components.

Reply:
and what's the reason for adding them here now?

Reply:
No reason, just wanted to move all the variables at once. But yeah, makes sense to do it step by step.

Reply:
@torkelo moved radio button vars to the component and removed from the theme. Will do the same for other components too. Until there's a need for exposing them for external usage I'll keep them private.